packages/analytics-js/src/components/eventManager/types.ts

```diff
@@ -28,8 +28,8 @@ export type APIEvent = {
   callback?: () => void;
   userId?: Nullable<string>;
   traits?: Nullable<ApiObject>;
-  to?: string;
-  from?: string;
+  to?: Nullable<string>;
+  from?: Nullable<string>;
```

Comment:
This should not be nullable as it'll always have some value from the analytics class layer (it is set as optional for the sake of other APIs).

Reply:
reverted